Update and complete Gdal warp algorithm description (#3754)
qgis/QGIS-Documentation: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst

  Default: *0*

``Georeferenced extents of output file to be created (xmin, xmax, ymin, ymax)`` [extent]
  Optional

>   Sets georeferenced extents of output file to be created (in ``target CRS`` by default or in ``CRS of the target raster extent`` if specified).

Reviewer: ```suggestion
  Sets georeferenced extent of the output file to be created (``target CRS`` by default. ``CRS of the target raster extent``, if specified).
```

Author: I'm not convinced by `(``target CRS`` by default. ``CRS of the target raster extent``, if specified)` . It looks like if the extent was the `target crs` though it's expressed `IN the target crs`. Then the full stop and sentence that follows look weird to me

Reviewer: My mistake - sorry about that!  I have tried to update.

``CRS of the target raster extent`` [crs]
  Optional

>   Specifies the CRS in which to interpret the coordinates given for the extents

Author: ```suggestion
  Specifies the CRS in which to interpret the coordinates given for the extent
```

>   of output file. This must not be confused with the target CRS of the output

Reviewer: ```suggestion
  of the output file. This must not be confused with the target CRS of the output
```

  dataset. It is instead a convenience e.g. when knowing the output coordinates

>   in a geodetic long/lat SRS, but still wanting a result in a projected coordinate system.

Reviewer: ```suggestion
  in a geodetic long/lat CRS, but wanting a result in a projected coordinate system.
```

``Use multithreaded warping implementation`` [boolean]

>   Two threads will be used to process chunks of image and perform input/output

Reviewer: ```suggestion
  Two threads will be used to process chunks of the image and perform input/output
```

>   operation simultaneously. Note that computation is not multithreaded itself.

Reviewer: ```suggestion
  operations simultaneously. Note that computation is not multithreaded itself.
```

  Default: *False*

``Additional command line parameters`` [string]
  Optional

>   Allows to add more advanced reprojection parameters.

Reviewer: ```suggestion
  Allows adding more advanced reprojection parameters.
```
